src/dagmc/dagmcmetadata.cpp

```diff
@@ -250,7 +254,15 @@ void dagmcMetaData::parse_importance_data() {
       imp_particles.insert(pair.first);
       // insert into map too
       if (importance_map[eh].count(pair.first) == 0) {
-        importance_map[eh][pair.first] = atof(pair.second.c_str());
+        char* e;
+        errno = 0;
+        double imp = std::strtod(pair.second.c_str(), &e);
+        if (*e != '\0' || errno != 0) {
+          std::cerr << "Can't parse " << pair.second << " as a float."
```

Comment:
Provide the user a little more insight about where this came up.
```suggestion
          std::cerr << "Can't parse importance " << pair.second << " as a float."
```